### PR TITLE
Remove dev install

### LINKED ISSUE #159: New ceph-deploy version breaks current dev install scripts
With the new ceph-deploy 2.0.0, they removed the options of both ceph-deploy osd prepare and ceph-deploy osd activate. This causes intall_ceph.sh to fail. We need to either install ceph-deploy 1.5.39 instead or modify the script to use ceph-deploy osd create. Quick fix for people who want to use the scripts right now is to change the lines in install_packages.sh "sudo pip install -U ceph-deploy" to be "sudo pip install -U ceph-deploy==1.5.39"

### PR DESCRIPTION
**Pull request by naved001**
We have the scripts to setup ceph and HIL on a different repo, and we have the ansible playbook here to deploy bmi. All that obviates the need for the scripts to setup a whole development environment.

Fixes #159

### CODE REVIEW

**Comment by naved001** [comment]
oh and yeah, it moves the installation stuff out of the scripts folder.

File: docs/Developer-guidlines.md [hunk 1]
```diff
@@ -20,6 +20,15 @@ If you're looking to code, there are a few ways to help:
 * Reviewing [Pull Requests](https://github.com/CCI-MOC/ims/pulls)
 * Discussions on open github issues. 
 
+
+## Settin up a dev environment
```
Comment (ianballou) [inline docs/Developer-guidlines.md]: I think you forgot a 'g' here

**Review by ianballou — APPROVED** [review]
All good, just a comment.

File: docs/Developer-guidlines.md (hunk 1 above)
Comment (naved001) [reply docs/Developer-guidlines.md]: Good catch, I'm talking like a G.

**Review by ianballou — APPROVED** [review]

File: docs/Developer-guidlines.md (hunk 1 above)
Comment (naved001) [reply docs/Developer-guidlines.md]: fixed.

**Comment by naved001** [comment]
@apoorvemohan @mihirborkar can either of you review this? This is pretty straightforward.

**Review by jeremyfreudberg — APPROVED** [review]
looks fine

**Pull request merged**